### ISSUE TITLE
Sliders are not presented when the screen size is small

### ISSUE THREAD

**Issue by eyalk11** [napari/superqt#50]
**Describe the bug**
The range sliders are not displayed sometimes and sometimes they do.

That is really a matter of proper width and height . I am not sure..

**To Reproduce**
This is not minimal, I admit. 
```
pip install git+https://github.com/eyalk11/compare-my-stocks
python -m compare_my_stocks 
```

Try on big screen , and on small, you wont. 
Also, I tried resizing the screen to smaller size, and increasing it and they disappeared. 

Given the same code, on a 21' screen it worked , and on 14' it didn't.

**Screenshots**
Screenshots and GIFS are much appreciated when reporting visual bugs.
![image](https://user-images.githubusercontent.com/72234965/147873897-245e7cb5-4efb-48bb-828c-e6ddbe44f8f9.png)

Following resize 

![image](https://user-images.githubusercontent.com/72234965/147873916-83503bfa-8fe0-4c72-807f-6d44f9c22929.png)

**Desktop (please complete the following information):**
 - OS with version Windows 10 
 - Qt Backend PySide6  Qt 6.2.0
 - Python version 3.9.6

**Comment by tlambert03** [comment]
thanks for the report @eyalk11.  will try to have a look soon.
(note, the 6.2.1.1 bug is now fixed as of #51)

**Comment by eyalk11** [comment]
I corrected the sizes a bit and it doesn't happen. 
There are two other bugs in this screenshot. The handles' values are behind the form's labels. and the handles are not the right shape.

**Comment by tlambert03** [comment]
yes, proper scaling on windows has generally been difficult, it seems to vary from PC to PC, depending on screen scaling and resolution.  it's not something I have particular expertise in, so welcome anyone who does to weigh in
